### PR TITLE
Profile molecule jobs

### PR DESCRIPTION
**Pull request by fao89**

### CODE REVIEW

**Comment by fao89** [comment]
recheck

File: ci/playbooks/molecule-test.yml [hunk 1]
```diff
@@ -12,6 +12,7 @@
     - name: Run molecule
       environment:
         ANSIBLE_LOG_PATH: "{{ ansible_user_dir }}/zuul-output/logs/ansible-execution.log"
+        ANSIBLE_CALLBACKS_ENABLED: "ansible.posix.profile_tasks,yaml"
```
[the remark this answers is not in the document]
Comment (fao89) [reply ci/playbooks/molecule-test.yml:15]: just wanted to have the same value as the current ansible.cfg: https://github.com/openstack-k8s-operators/ci-framework/blob/main/ansible.cfg#L8

[the remark this answers is not in the document]
Comment (pablintino) [reply ci/playbooks/molecule-test.yml:15]: Cool, lgtm

**Review by rebtoor — COMMENTED** [review]
/lgtm

**Pull request merged**